submissions/israelOladosu/DomManipulation/Script.js

```diff
@@ -0,0 +1,107 @@
+let submitForm = document.getElementById("validateForm"),
+  email = document.getElementById("email"),
+  password = document.getElementById("password"),
+  confirmPassword = document.getElementById("pswd"),
+  firstName = document.getElementById("firstName"),
+  lastName = document.getElementById("lastName"),
+  otherName = document.getElementById("otherName"),
+  phoneNumber = document.getElementById("phone");
+
+var regEx = {
+  name: /[^A-Za-z]/,
+  password: /^(?=.*[A-Z])(?=.*[a-z])(?=.*[@$!%*?&_<>':;])(?=.*[\d])[a-zA-Z0-9@$!%*?&_<>':;]{6,}$/,
+  email: /^([a-z\d\.-]+)@([a-z\d-]+)\.([a-z]{2,8})$/i,
+  phoneNumber: /^[\+0-9]{11,}$/,
+};
+
+function required(field, e) {
+  if (field.id == "pswd" && field.value == "") {
```

Comment:
Why are you doing this check `field.id == "pswd"`?

Reply:
Beacause the error message of confirm password is quite different from the format of other fields... I am grabing the label input text to output the error but the label of confirm password does not deliver the message stated in the success criteria, so i had to specailly write it out

Reply:
Then I will suggest to pass an extra parameter to the function, say `error message`

Reply:
Thank you, I have done as you have suggested.